libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -1363,6 +1369,12 @@ export const appStrings = {
     </UiString>
   ),
 
+  warningCvrsNeedSync: () => (
+    <UiString uiStringKey="warningCvrsNeedSync">
+      A poll worker must sync Cast vote records (CVRs) to the USB drive.
+    </UiString>
+  ),
```

Comment:
> I am open to making the copy in the potentially voter facing first screen more generic, I primarily copied how the ballot bag full screen works which does give that context.

Yeah I'm a bit torn here since "CVRs need to be synced" is more in the weeds than "the ballot bag is full". But ultimately, I like the consistency with the ballot bag screen. And I think the immediate context for poll workers, before card insertion, is nice. (I know poll workers won't be able to read the initial message if a voter has changed the language, and the poll worker doesn't speak that language, but we don't expect voters to change the language on this screen often.)

+1 to Roe's suggestion to lower-case the C in cast vote records